src/main/java/edu/harvard/iq/dataverse/authorization/providers/builtin/DataverseUserPage.java

```diff
@@ -284,6 +284,12 @@ public void validateNewPassword(FacesContext context, UIComponent toValidate, Ob
 
     public String save() {
         boolean passwordChanged = false;
+        
+        //First reget user to make sure they weren't deactivated or deleted
+        if (session.getUser().isAuthenticated() && !session.getUser(true).isAuthenticated()) {
```

Comment:
is the first part of this needed?

Reply:
I guess not. it's left over from when I did the re-get user within the save method.

Reply:
no. we need it because if we're creating an account the session user is a guest before they save the new account.